geomesa-security/src/main/scala/org/locationtech/geomesa/security/NoopAuditProvider.scala

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2013-2015 Commonwealth Computer Research, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, Version 2.0 which
+ * accompanies this distribution and is available at
+ * http://www.opensource.org/licenses/apache2.0.php.
+ */
+
+package org.locationtech.geomesa.security
+
+import java.io.Serializable
+import java.util.{Map => jMap}
+
+import scala.collection.JavaConversions._
+
+object NoopAuditProvider extends AuditProvider {
+
+  override val getCurrentUserId: String = "unknown"
```

Comment:
I'm changing it to use the value from the data store params if nothing else is provided. Note that I provided a spring security impl in geomesa-plugin so that in geoserver it will hook into geoserver's spring security by default, and so you'll get the logged in user.